frontend/public/style/ancillary/_bootstrap-residual.scss

```diff
@@ -109,5 +109,5 @@
 
 .small,
```

Comment:
can we use the `pf-v6-u-font-size-xs` class here and remove styling for `.small`?

Reply:
We could, but we need to leave the declaration for `<small>`, so I opted to leave `.small` as well.